examples/next-13/README.md

```diff
@@ -0,0 +1,3 @@
+# Livepeer Player
```

Comment:
**nit:** Seems confusing to me that the livepeer player is "a next 13 example". It could certainly serve as an example but that's not what it was built for, nor its main use case.

So IMO it'd make more sense to have this as a separate package in the `packages` folder, also closely tied to the main development process here, like later we can create some tests to make sure it doesn't break between SDK versions, etc. Then here, we can reference it as an example, maybe only with a readme on `examples/next-13` or sth like that? WDYT?